docs/modules/components/pages/processors/ollama_embeddings.adoc

```diff
@@ -38,9 +38,14 @@ Common::
 # Common config fields, showing default values
 label: ""
 ollama_embeddings:
-  server_address: http://127.0.0.1:11434 # No default (optional)
   model: nomic-embed-text # No default (required)
   text: "" # No default (optional)
+  max_tokens: 0 # No default (optional)
```

Comment:
See comments from Ollama chat page - above. It looks like all the fields/options are the same?